kms/spec/kms_spec.rb

```diff
@@ -0,0 +1,367 @@
+        location: @location)
+
+    expect(after_version_list.total_size).to be > before_version_list.total_size
+  end
+
+  it "can disable a cryptoKey version" do
+    test_cryptokey_id = "#{@project_id}-disable-#{Time.now.to_i}"
+
+    cryptokey = test_create_cryptokey(project_id: @project_id,
+        key_ring_id: @key_ring_id, crypto_key: test_cryptokey_id,
+        location: @location)
+
+    version = "1" # first version is labeled 1
+
+    expect {
+      $disable_cryptokey_version.call(project_id: @project_id,
+         key_ring_id: @key_ring_id, crypto_key: test_cryptokey_id,
+         version: version, location: @location)
+    }.to output(/Disabled version #{version} of #{test_cryptokey_id}/).to_stdout
+
+    cryptokey = test_get_cryptokey_version(project_id: @project_id,
+        key_ring_id: @key_ring_id, crypto_key: test_cryptokey_id,
+        version: version, location: @location)
+
+    expect(cryptokey.state).to eq "DISABLED"
+  end
+
+  it "can destroy a cryptoKey version" do
+    test_cryptokey_id = "#{@project_id}-destroy-#{Time.now.to_i}"
+
+    cryptokey = test_create_cryptokey(project_id: @project_id,
+        key_ring_id: @key_ring_id, crypto_key: test_cryptokey_id,
+        location: @location)
+
+    version = "1" # first version is labeled 1
+
+    expect {
+      $destroy_cryptokey_version.call(project_id: @project_id,
+         key_ring_id: @key_ring_id, crypto_key: test_cryptokey_id,
+         version: version, location: @location)
+    }.to output(/Destroyed version #{version} of #{test_cryptokey_id}/).to_stdout
+
+    cryptokey = test_get_cryptokey_version(project_id: @project_id,
+        key_ring_id: @key_ring_id, crypto_key: test_cryptokey_id,
+        version: version, location: @location)
+
+    expect(cryptokey.state).to eq "DESTROY_SCHEDULED"
+  end
+
+  it "can add a member to a cryptokey policy" do
+    expect {
+      $add_member_to_cryptokey_policy.call(project_id: @project_id,
+          key_ring_id: @key_ring_id, crypto_key: @cryptokey_id,
+          member: "user:test@test.com", role: "roles/owner",
+          location: @location)
+    }.to output(/test@test.com/).to_stdout
+
+    policy = test_get_cryptokey_policy(project_id: @project_id,
+          key_ring_id: @key_ring_id, crypto_key: @cryptokey_id,
+          location: @location)
+
+    members = policy.bindings.map(&:members).flatten
+
+    expect(members).to include("user:test@test.com")
+  end
+
+  it "can get a keyring policy" do
+    test_add_member_to_keyring_policy(project_id: @project_id,
+          key_ring_id: @key_ring_id, member: "serviceAccount:test-account@cloud-samples-tests-ruby.iam.gserviceaccount.com",
+          role: "roles/owner", location: @location)
+
+    expect {
+      $get_keyring_policy.call(project_id: @project_id, key_ring_id: @key_ring_id,
+          location: @location)
+    }.to output(/serviceAccount:test-account@cloud-samples-tests-ruby.iam.gserviceaccount.com/).to_stdout
```

Comment:
I think this should be `"serviceAccount:test-account@#{@project_id}-ruby.iam.gserviceaccount.com"`